src/test/java/reposense/parser/RepoConfigurationTest.java

```diff
@@ -55,13 +55,22 @@
     private static final List<String> SECOND_AUTHOR_GLOB_LIST = Arrays.asList("**[!(.md)]", "collated**");
     private static final List<String> THIRD_AUTHOR_GLOB_LIST = Arrays.asList("", "collated**");
 
-    private static final List<String> CONFIG_FORMATS = Arrays.asList("java", "adoc", "md");
+    private static final List<Format> CONFIG_FORMATS = Format.convertStringsToFormats(Arrays.asList(
+            "java", "adoc", "md"));
     private static final List<String> CLI_FORMATS = Arrays.asList("css", "html");
 
     private static RepoConfiguration REPO_DELTA_STANDALONE_CONFIG;
 
+    static {
+        try {
+            TEST_REPO_DELTA = new Location("https://github.com/reposense/testrepo-Delta.git");
+        } catch (InvalidLocationException e) {
```

Comment:
My intention of creating `TEST_REPO_DELTA` was to simplify tests.

`i.e.` `Assert.assertEquals(new Location(TEST_REPO_BETA_LOCATION), config.getLocation());` -> `Assert.assertEquals(TEST_REPO_BETA, config.getLocation());`

However, I have overlooked that an ugly `try-catch` block is required. 

@yamidark 
What do you think of this, should we stick the to previous way?
`Assert.assertEquals(new Location(TEST_REPO_BETA_LOCATION), config.getLocation());`
Since there are some tests that still haven't got refactored.